tjreports/administrator/views/tjreport/view.html.php

```diff
@@ -111,8 +111,8 @@ protected function addDocumentHeaderData()
 
 		$document = Factory::getDocument();
 
-		HTMLHelper::_('script', 'components/com_tjreports/assets/js/tjrContentService.min.js');
-		HTMLHelper::_('script', 'components/com_tjreports/assets/js/tjrContentUI.min.js');
+		HTMLHelper::_('script', 'com_tjreports/tjrContentService.min.js', array("relative" => true));
```

Comment:
hope you did changes in xml accordingly and tested installation too @divyachaudhari